### ISSUE TITLE
Possible to set frame of the window in slate.js?

### ISSUE THREAD

**Issue by WarWithinMe** [jigish/slate#327]
Right now using window.move() and window.resize() to manipulate the rect of the window causes the window to move then suddenly jump to new size.

I wonder if there's a way to call window.setFrame directly. Is it possible to use `NSAccessibilityWindowAttribute` to get do that?

**Comment by sdegutis** [comment]
@WarWithinMe No, the only two that can be used are `NSAccessibilitySizeAttribute` and `NSAccessibilityPositionAttribute`. In [Zephyros](https://github.com/sdegutis/zephyros) I have a function to set the frame of a window, but it technically just sets the size, then the position, then the size again. The extra set-size is needed because sometimes you can't move the window to the right position until it's smaller, and sometimes you can't resize the window to the new size until it's in the right position. So you can just use this technique to get the same results.
